### PR TITLE
RET-881 Current Work Address

### PR DESCRIPTION
**Pull request by brendanmchugh**
### JIRA link

https://tools.hmcts.net/jira/browse/RET-881

### Change description

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/test/routes/work-address.test.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import request from 'supertest';
+
+import { app } from '../../main/app';
+
+const PAGE_URL = '/work-address';
```
Comment (lvo700) [inline src/test/routes/work-address.test.ts:5]: Could import PageUrls for this?

File: src/test/unit/views/work-address.test.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import { expect } from 'chai';
+import request from 'supertest';
+
+import { app } from '../../../main/app';
+
+const PAGE_URL = '/work-address';
```
Comment (lvo700) [inline src/test/unit/views/work-address.test.ts:6]: Same as above

File: src/test/routes/work-address.test.ts (hunk 1 above)
Comment (brendanmchugh) [reply src/test/routes/work-address.test.ts]: Thanks I've deleted this route, controller and tests, reusing place-of-work for this story instead.

File: src/test/unit/views/work-address.test.ts (hunk 2 above)
Comment (brendanmchugh) [reply src/test/unit/views/work-address.test.ts]: Thanks.

**Review by lvo700 — APPROVED** [review]

**Review by pmumbere — APPROVED** [review]

**Pull request merged**